build_docs.py

```diff
@@ -242,6 +242,21 @@ def run(cmd, cwd=None) -> subprocess.CompletedProcess:
     return result
 
 
+def run_with_logging(cmd, cwd=None):
+    """Like subprocess.check_call, with logging before the command execution."""
+    cmd = list(map(str, cmd))
+    logging.debug("Run: %s", shlex.join(cmd))
```

Comment:
A bit late to the party, but I noticed some inconsistency between this new function and the `run` function defined above:
1. `run` uses `"Run: '%s'"` (with quotes around the `%s`), whereas `run_with_logging` uses `"Run: %s"` (without quotes);
2. in `run`, `"Run: %r"` was changed to `"Run: '%s'"`, but there's still a `"Run: %r KO:\n%s"` a few lines after that;
3. `run_with_logging` uses `list` + `map` instead of using a listcomp (both are ok, but it would be better if they were the same)

Reply:
See #213